### PR TITLE
Automatic build

### PR DESCRIPTION
**Pull request by hbons**

### CODE REVIEW

File: samples/FigmaSharp/StoryboardRendering/StoryboardRendering.Cocoa/StoryboardRendering.Cocoa.csproj [hunk 1]
```diff
@@ -9,7 +9,8 @@
     <RootNamespace>NativeControls.Cocoa</RootNamespace>
     <AssemblyName>NativeControls.Cocoa</AssemblyName>
      <MonoMacResourcePrefix>Resources</MonoMacResourcePrefix>
-     <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (netonjm) [inline samples/FigmaSharp/StoryboardRendering/StoryboardRendering.Cocoa/StoryboardRendering.Cocoa.csproj]: this need to be reverted

Comment (netonjm) [inline samples/FigmaSharp/StoryboardRendering/StoryboardRendering.Cocoa/StoryboardRendering.Cocoa.csproj]: this need to be reverted

File: samples/FigmaSharp/ToCode/ToCode.Cocoa/ToCode.Cocoa.csproj [hunk 2]
```diff
@@ -8,8 +8,9 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>ToCode.Cocoa</RootNamespace>
     <AssemblyName>ToCode.Cocoa</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (netonjm) [inline samples/FigmaSharp/ToCode/ToCode.Cocoa/ToCode.Cocoa.csproj]: this need to be reverted

File: samples/FigmaSharp/MainSample/MainSample.Cocoa/Views/DocumentViewController.cs [hunk 3]
```diff
@@ -50,28 +50,29 @@ public partial class DocumentViewController : NSViewController
 
 		DocumentWindowController windowController;
 
-		public DocumentViewController (IntPtr handle) : base (handle)
```
Comment (netonjm) [inline samples/FigmaSharp/MainSample/MainSample.Cocoa/Views/DocumentViewController.cs]: it seems you didn't configured in your VS some code guidelines and broken when you formatting the document, do we want to follow monodevelop one?
https://www.mono-project.com/community/contributing/coding-guidelines/

File: FigmaSharp.Views/FigmaSharp.Views.Cocoa/FigmaSharp.Views.Cocoa.csproj [hunk 4]
```diff
@@ -8,8 +8,9 @@
     <OutputType>Library</OutputType>
     <RootNamespace>FigmaSharp.Views.Cocoa</RootNamespace>
     <AssemblyName>FigmaSharp.Views.Cocoa</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (netonjm) [inline FigmaSharp.Views/FigmaSharp.Views.Cocoa/FigmaSharp.Views.Cocoa.csproj]: this needs to be reverted

**Comment by hbons** [comment]
I'll make a new PR.

**Pull request closed without merging**